dp-identity-data-migration/userdataextraction/user_extraction.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"strings"
+
+	"os"
+
+	"time"
+
+	"github.com/ONSdigital/dp-zebedee-sdk-go/zebedee"
+	"github.com/google/uuid"
+)
+
+type config struct {
+	filename,
+	host,
+	pword,
+	user string
+}
+
+func readConfig() (conf config) {
```

Comment:
Create all required `vars/types` at the start of the package and then functions beneath
```suggestion
type config struct {}

type cognito_user struct {}

var header = cognito_user{}

func readConfig() (conf config) {...

```